x-pack/platform/plugins/shared/onechat/public/application/components/conversations/new_conversation_prompt.tsx

```diff
@@ -66,45 +65,203 @@ export const NewConversationPrompt: React.FC<{}> = () => {
     ),
   };
   return (
-    <ConversationContent css={fullHeightStyles}>
-      <EuiFlexGroup
-        css={promptStyles}
-        direction="column"
-        alignItems="center"
-        justifyContent="center"
-        aria-label={labels.container}
-      >
-        <EuiFlexItem grow={false}>
-          <EuiIcon color="primary" size="xxl" type="logoElastic" />
-        </EuiFlexItem>
-        <EuiFlexItem>
-          <EuiTitle>
-            <h2>{labels.title}</h2>
-          </EuiTitle>
-        </EuiFlexItem>
-        <EuiFlexItem>
-          <EuiText textAlign="center" color="subdued">
-            <p>{labels.subtitle}</p>
-          </EuiText>
-        </EuiFlexItem>
-        <EuiFlexItem grow={false}>
-          <EuiButton
-            href="#"
-            target="_blank"
-            size="m"
-            aria-label={i18n.translate(
-              'xpack.onechat.newConversationPrompt.agentBuilderDocsAriaLabel',
-              {
-                defaultMessage: 'Read Agent Builder documentation',
+    <EuiFlexGroup
```

Comment:
Hmm can't we use https://eui.elastic.co/docs/components/display/empty-prompt/  here?
 
I believe this is default style for empty state pages in EUI cc @julianrosado